sample-submodule/build.gradle

```diff
@@ -3,14 +3,14 @@ plugins {
     id 'kotlin-android'
 }
 
-if (project.hasProperty('useKsp')) {
+//if (project.hasProperty('useKsp')) {
     apply plugin: 'com.google.devtools.ksp'
-} else {
-    apply plugin: 'kotlin-kapt'
-    kapt {
-        correctErrorTypes = true
-    }
-}
+//} else {
+//    apply plugin: 'kotlin-kapt'
+//    kapt {
+//        correctErrorTypes = true
+//    }
+//}
```

Comment:
Will revert these changes. It was just to make it easier to test locally. Sorry about this